CourseStream/CourseStream.Data/CourseStreamDBContext.cs

```diff
@@ -1,22 +1,13 @@
 ﻿using Microsoft.EntityFrameworkCore;
 namespace CourseStream.Data;
 
-public class CourseStreamDBContext : DbContext
+public class CourseStreamDbContext : DbContext
 {
-    public DbSet<CourseStream> CourseStreams { get; set; }
-    public DbSet<Enrollment> Enrollments { get; set; }
-    public CourseStreamDBContext()
-    {
-        
-    }
-    public CourseStreamDBContext(DbContextOptions<CourseStreamDBContext> options) : base(options)
-    {
-    }
-
-    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```

Comment:
Same as prev. How this will affect on our current migrations applying flow?

Reply:
This should be deleted in order to launch correctly application in docker. All configuration are done in startup